packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```diff
@@ -0,0 +1,366 @@
+    "formSubmissionSuccessful.textMessage": "Si un compte est associé à {email}, vous recevrez un e-mail avec un lien pour changer votre mot de passe.",
+    "giftCard.balance": "Équilibre: ",
+    "giftCard.remove": "Retirer",
+    "giftCards.apply": "Appliquer",
+    "giftCards.balance": "Équilibre: ",
+    "giftCards.cardEntry": "Entrez le numéro de carte",
+    "giftCards.cardNumber": "Numéro de carte-cadeau",
+    "giftCards.checkBalance": "Vérifier le solde",
+    "giftCards.errorInvalid": "Carte invalide. Veuillez réessayer.",
+    "giftCards.errorLoading": "Une erreur s'est produite lors du chargement des cartes-cadeaux appliquées. Veuillez actualiser et réessayer.",
+    "giftCards.errorRemoving": "Impossible de retirer la carte-cadeau. Veuillez réessayer.",
+    "giftCards.loading": "Chargement des cartes-cadeaux ...",
+    "giftCardSection.giftCard": "Appliquer des chèques cadeaux",
+    "giftCardSummary.lineItemLabel": "Carte(s) cadeau appliquée",
+    "giftOptions.cardMessage": "Entrez votre message ici",
+    "giftOptions.includeGiftReceipt": "Inclure le reçu cadeau",
+    "giftOptions.includePrintedCard": "Inclure la carte imprimée",
+    "Give Back": "Redonner",
+    "global.addButton": "Ajouter",
+    "global.cancelButton": "Annuler",
+    "global.changePassword": "Changer le mot de passe",
+    "global.city": "Ville",
+    "global.confirmButton": "Confirmer",
+    "global.creditCard": "Carte de crédit",
+    "global.currentPassword": "Mot de passe actuel",
+    "global.editButton": "Éditer",
+    "global.email": "Courriel",
+    "global.firstName": "Prénom",
+    "global.free": "Gratuite",
+    "global.home": "Accueil",
+    "global.lastName": "Nom De Famille",
+    "global.name": "Nom",
+    "global.newPassword": "nouveau mot de passe",
+    "global.password": "Mot De Passe",
+    "global.phoneNumber": "Numéro de téléphone",
+    "global.postcode": "ZIP / Code Postal",
+    "global.private": "Privée",
+    "global.public": "Publique",
+    "global.quantity": "Quantité",
+    "global.save": "Sauvegarder",
+    "global.saveAndContinueButton": "Sauvegarder et continuer",
+    "global.shippingInformation": "Informations Sur La Livraison",
+    "global.shippingMethod": "Mode de livraison",
+    "global.sku": "FR-SKU",
+    "global.streetAddress": "Adresse de rue",
+    "global.streetAddress2": "Adresse postale 2",
+    "global.updateButton": "Mise à jour",
+    "guestForm.continueToNextStep": "Continuer vers la méthode d'expédition",
+    "guestForm.emailMessage": "Définissez un mot de passe à la fin du paiement des invités pour créer un compte en une seule étape.",
+    "Help": "Aidez-moi",
+    "Live Chat": "Chat en direct",
+    "loadingIndicator.message": "Récupération des données...",
+    "logo.title": "Venia",
+    "magentoRoute.routeError": "Cette page est introuvable. Veuillez réessayer.",
+    "magentoRoute.internalError": "Un problème est survenu. Veuillez réessayer.",
+    "miniCart.checkout": "CHECK-OUT",
```

Comment:
This seems like a weird translation. There is no "Check-out" in french. I'm not sure what is the intend of this label. Used on a button to go to checkout?

Reply:
Used for the button in the mini cart that takes the shopper to the cart/checkout page.
[![Image from Gyazo](https://i.gyazo.com/d35f08f43d982c23e58e3a7031925c8c.png)](https://gyazo.com/d35f08f43d982c23e58e3a7031925c8c)

Reply:
@sirugh This is actually a good dilemma as there is no literal translation for Check-Out. I looked multiple french websites and asked colleagues and here's a couple suggestions : 
"Paiement" (my favorite for a small button, could also be used as a title for checkoutPage.titleCheckout)
"Procéder au paiement"
"Passer à la caisse"